plugins/chat/assets/stylesheets/common/chat-message-actions.scss

```diff
@@ -60,11 +60,23 @@
     &:first-child {
       border-bottom-left-radius: 0.25em;
       border-top-left-radius: 0.25em;
+
+      .rtl & {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0.25em;
+        border-top-left-radius: 0;
+        border-top-right-radius: 0.25em;
+      }
     }
 
     &:first-child:not(:hover) {
       border-color: var(--primary-300);
       border-right-color: transparent;
+
+      .rtl & {
+        border-color: var(--primary-300);
```

Comment:
I don't think we need to repeat this here

Reply:
@jjaffeux, thank you for your review. Actually, I believe it is necessary to include this line because it "unsets" the `border-right-color` in RTL mode. Here is a visual comparison of how it appears with and without this line:

![image](https://github.com/discourse/discourse/assets/35470921/435bf2c7-b11c-4e8c-a6a8-519acf153e64)

As you can see, when we omit the `border-right-color: border-color: var(--primary-300);` part, the right border of the block is missing. Including this line ensures that the right border is properly rendered in the desired manner. What do you think?

Reply:
Oh yes sorry I actually missed the `border-right-color: transparent;` line, works for me.